libs/core/langchain_core/messages/base.py

```diff
@@ -95,6 +95,38 @@ def merge_content(
         else:
             return_list: List[Union[str, Dict]] = [first_content]
             return return_list + second_content
+    elif isinstance(second_content, dict) and isinstance(
```

Comment:
contents are only supposed to be strs or lists, no?

Reply:
No? `{"type":"text","text":" San"}`